cisco_secure_email_threat_defense/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "9776e6c8-2031-4dda-98b5-3628b181625b",
+  "app_id": "cisco-secure-email-threat-defense",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Gain insights into Cisco Secure Email Threat Defense logs",
+    "title": "Cisco Secure Email Threat Defense",
+    "media": [],
+    "classifier_tags": [
+      "Category::Log Collection",
+      "Category::Security",
+      "Submitted Data Type::Logs",
+      "Offering::Integration"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "auto_install": false,
+      "source_type_id": 21617758,
+      "source_type_name": "Cisco Secure Email Threat Defense",
```

Comment:
`cisco-secure-email-threat-defense`

Reply:
@Wyrine Previously for Cisco Secure Endpoint, we have received comment to to keep source_type_name as natural title-cased. Can you please let us know which one to keep?

Link: https://github.com/DataDog/integrations-core/pull/17865#discussion_r1666827915 
Screenshot: 
![image](https://github.com/user-attachments/assets/e32042ba-0b22-472a-a522-427ad4733533)